lib/membrane_opus/encoder.ex

```diff
@@ -173,7 +171,29 @@ defmodule Membrane.Opus.Encoder do
   end
 
   defp set_current_pts(%{queue: <<>>} = state, input_pts) do
-    %{state | current_pts: input_pts}
+    if state.current_pts != nil and input_pts != nil and state.current_pts > input_pts do
+      diff = (state.current_pts - input_pts) |> Time.nanoseconds()
```

Comment:
if `state.current_pts` and `input_pts` are `Membrane.Time.t()`, there is no need to pass the difference between them to `Membrane.Time.nanoseconds/1`